### ISSUE TITLE
[Issue] Missing `icudtl.dat` on Linux CEF Engine

### ISSUE THREAD

**Issue by BrandonMayjeye** [Voltstro-Studios/UnityWebBrowser#346]
### What platform are you experiencing this issue on?

Linux x64

### What version of UWB are you using?

2.2.0

### What Unity version are you running?

2022.3.48f1

### Describe what the issue you are experiencing is.

The browser never starts, or runs. 
UWB stalls waiting for engine and then gives 
[UWB]: The engine did not get ready within engine startup timeout! The engine process is not even running! Exit code: 133.
UnityEngine.Logger:LogError (string,object)

maybe related to 
https://github.com/Voltstro-Studios/UnityWebBrowser/issues/166

### Provide reproducible steps for this issue.

Followed guid from repo
Setup Volt 
Installed with scopes.
Got UnityWebBrowser installed
set engine startup to 70,000
set engine,input handler, communication layer
![image](https://github.com/user-attachments/assets/de9d2562-e4ba-4ded-b7b2-01cd44c3dfd8)

No processes on computer called UWB , CEF,  or Chromium running 

File @ /home/brandon/Documents/Mayjeye/UnderworldHustle/Library/PackageCache/dev.voltstro.unitywebbrowser.engine.cef.linux.x64@2.2.0-128.4.9/Engine~/UnityWebBrowser.Engine.Cef
Exists!

### Any additional info you like to provide?

Logs From Unity
```


[UWB]: Starting browser engine process from '/home/brandon/Documents/Mayjeye/UnderworldHustle/Library/PackageCache/dev.voltstro.unitywebbrowser.engine.cef.linux.x64@2.2.0-128.4.9/Engine~/UnityWebBrowser.Engine.Cef'...
UnityEngine.StackTraceUtility:ExtractStackTrace () (at /home/bokken/build/output/unity/unity/Runtime/Export/Scripting/StackTrace.cs:37)
UnityEngine.DebugLogHandler:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[])
UnityEngine.Logger:Log (UnityEngine.LogType,string,object)
VoltstroStudios.UnityWebBrowser.Logging.DefaultUnityWebBrowserLogger:Debug (object) (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs:26)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient:Init () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:296)
VoltstroStudios.UnityWebBrowser.Core.BaseUwbClientManager:Start () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/BaseUwbClientManager.cs:32)

(Filename: ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs Line: 26)

[UWB]: Using communication layer of 'TCP'.
UnityEngine.StackTraceUtility:ExtractStackTrace () (at /home/bokken/build/output/unity/unity/Runtime/Export/Scripting/StackTrace.cs:37)
UnityEngine.DebugLogHandler:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[])
UnityEngine.Logger:Log (UnityEngine.LogType,string,object)
VoltstroStudios.UnityWebBrowser.Logging.DefaultUnityWebBrowserLogger:Debug (object) (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs:26)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient:Init () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:347)
VoltstroStudios.UnityWebBrowser.Core.BaseUwbClientManager:Start () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/BaseUwbClientManager.cs:32)

(Filename: ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs Line: 26)

[UWB]: Process Path: '/home/brandon/Documents/Mayjeye/UnderworldHustle/Library/PackageCache/dev.voltstro.unitywebbrowser.engine.cef.linux.x64@2.2.0-128.4.9/Engine~/UnityWebBrowser.Engine.Cef'
Working: '/home/brandon/Documents/Mayjeye/UnderworldHustle/Library/PackageCache/dev.voltstro.unitywebbrowser.engine.cef.linux.x64@2.2.0-128.4.9/Engine~/'
UnityEngine.StackTraceUtility:ExtractStackTrace () (at /home/bokken/build/output/unity/unity/Runtime/Export/Scripting/StackTrace.cs:37)
UnityEngine.DebugLogHandler:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[])
UnityEngine.Logger:Log (UnityEngine.LogType,string,object)
VoltstroStudios.UnityWebBrowser.Logging.DefaultUnityWebBrowserLogger:Debug (object) (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs:26)
VoltstroStudios.UnityWebBrowser.Core.Engines.EngineProcess:StartProcess (string,System.Diagnostics.DataReceivedEventHandler,System.Diagnostics.DataReceivedEventHandler) (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/Engines/EngineProcess.cs:66)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient:StartEngineProcess (string) (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:429)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient/<>c__DisplayClass61_0:<Init>b__0 () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:415)
Cysharp.Threading.Tasks.UniTask:Create (System.Func`1<Cysharp.Threading.Tasks.UniTask>) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.Factory.cs:81)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient:Init () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:414)
VoltstroStudios.UnityWebBrowser.Core.BaseUwbClientManager:Start () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/BaseUwbClientManager.cs:32)

(Filename: ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs Line: 26)

[UWB]: Arguments: '-initial-url "https://voltstro.dev" -width 1920 -height 1080 -javascript True -background-color FFFFFFFF -log-path "/home/brandon/Documents/Mayjeye/UnderworldHustle/Library/UnityWebBrowser.Engine.Cef.log" -log-severity Debug -comms-layer-name "TCP" -in-location "5555" -out-location "5556" -popup-action "Ignore" -local-storage True -proxy-server False'
UnityEngine.StackTraceUtility:ExtractStackTrace () (at /home/bokken/build/output/unity/unity/Runtime/Export/Scripting/StackTrace.cs:37)
UnityEngine.DebugLogHandler:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[])
UnityEngine.Logger:Log (UnityEngine.LogType,string,object)
VoltstroStudios.UnityWebBrowser.Logging.DefaultUnityWebBrowserLogger:Debug (object) (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs:26)
VoltstroStudios.UnityWebBrowser.Core.Engines.EngineProcess:StartProcess (string,System.Diagnostics.DataReceivedEventHandler,System.Diagnostics.DataReceivedEventHandler) (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/Engines/EngineProcess.cs:67)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient:StartEngineProcess (string) (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:429)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient/<>c__DisplayClass61_0:<Init>b__0 () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:415)
Cysharp.Threading.Tasks.UniTask:Create (System.Func`1<Cysharp.Threading.Tasks.UniTask>) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.Factory.cs:81)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient:Init () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:414)
VoltstroStudios.UnityWebBrowser.Core.BaseUwbClientManager:Start () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/BaseUwbClientManager.cs:32)

(Filename: ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs Line: 26)

[PhysX] Initialized MultithreadedTaskDispatcher with 24 workers.
[PhysX] Initialized MultithreadedTaskDispatcher with 24 workers.
TrimDiskCacheJob: Current cache size 0mb
[UWB]: The engine did not get ready within engine startup timeout! The engine process is not even running! Exit code: 133.
UnityEngine.StackTraceUtility:ExtractStackTrace () (at /home/bokken/build/output/unity/unity/Runtime/Export/Scripting/StackTrace.cs:37)
UnityEngine.DebugLogHandler:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[])
UnityEngine.Logger:LogError (string,object)
VoltstroStudios.UnityWebBrowser.Logging.DefaultUnityWebBrowserLogger:Error (object) (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs:36)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient/<WaitForEngineReadyTask>d__63:MoveNext () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:454)
Cysharp.Threading.Tasks.CompilerServices.AsyncUniTask`1<VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient/<WaitForEngineReadyTask>d__63>:Run () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/CompilerServices/StateMachineRunner.cs:189)
Cysharp.Threading.Tasks.AwaiterActions:Continuation (object) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.cs:25)
Cysharp.Threading.Tasks.UniTaskCompletionSourceCore`1<Cysharp.Threading.Tasks.AsyncUnit>:TrySetException (System.Exception) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTaskCompletionSource.cs:167)
Cysharp.Threading.Tasks.CompilerServices.AsyncUniTask`1<Cysharp.Threading.Tasks.UniTaskExtensions/<Timeout>d__44>:SetException (System.Exception) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/CompilerServices/StateMachineRunner.cs:210)
Cysharp.Threading.Tasks.CompilerServices.AsyncUniTaskMethodBuilder:SetException (System.Exception) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/CompilerServices/AsyncUniTaskMethodBuilder.cs:59)
Cysharp.Threading.Tasks.UniTaskExtensions/<Timeout>d__44:MoveNext () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTaskExtensions.cs:410)
Cysharp.Threading.Tasks.CompilerServices.AsyncUniTask`1<Cysharp.Threading.Tasks.UniTaskExtensions/<Timeout>d__44>:Run () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/CompilerServices/StateMachineRunner.cs:189)
Cysharp.Threading.Tasks.AwaiterActions:Continuation (object) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.cs:25)
Cysharp.Threading.Tasks.UniTaskCompletionSourceCore`1<System.ValueTuple`3<int, bool, bool>>:TrySetResult (System.ValueTuple`3<int, bool, bool>) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTaskCompletionSource.cs:139)
Cysharp.Threading.Tasks.UniTask/WhenAnyPromise`2<bool, bool>:TryInvokeContinuationT2 (Cysharp.Threading.Tasks.UniTask/WhenAnyPromise`2<bool, bool>,Cysharp.Threading.Tasks.UniTask`1/Awaiter<bool>&) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.WhenAny.Generated.cs:99)
Cysharp.Threading.Tasks.UniTask/WhenAnyPromise`2/<>c<bool, bool>:<.ctor>b__2_1 (object) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.WhenAny.Generated.cs:58)
Cysharp.Threading.Tasks.UniTaskCompletionSourceCore`1<object>:TrySetResult (object) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTaskCompletionSource.cs:139)
Cysharp.Threading.Tasks.UniTask/DelayPromise:MoveNext () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.Delay.cs:790)
Cysharp.Threading.Tasks.Internal.PlayerLoopRunner:RunCore () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/Internal/PlayerLoopRunner.cs:175)
Cysharp.Threading.Tasks.Internal.PlayerLoopRunner:Update () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/Internal/PlayerLoopRunner.cs:145)
Cysharp.Threading.Tasks.Internal.PlayerLoopRunner:Run () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/Internal/PlayerLoopRunner.cs:104)

(Filename: ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs Line: 36)

[UWB]: UWB shutdown...
UnityEngine.StackTraceUtility:ExtractStackTrace () (at /home/bokken/build/output/unity/unity/Runtime/Export/Scripting/StackTrace.cs:37)
UnityEngine.DebugLogHandler:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[])
UnityEngine.Logger:Log (UnityEngine.LogType,string,object)
VoltstroStudios.UnityWebBrowser.Logging.DefaultUnityWebBrowserLogger:Debug (object) (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Logging/DefaultUnityWebBrowserLogger.cs:26)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient:ReleaseResources () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:1091)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient:Dispose () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:1081)
VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient/<WaitForEngineReadyTask>d__63:MoveNext () (at ./Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.0/Runtime/Core/WebBrowserClient.cs:459)
Cysharp.Threading.Tasks.CompilerServices.AsyncUniTask`1<VoltstroStudios.UnityWebBrowser.Core.WebBrowserClient/<WaitForEngineReadyTask>d__63>:Run () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/CompilerServices/StateMachineRunner.cs:189)
Cysharp.Threading.Tasks.AwaiterActions:Continuation (object) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.cs:25)
Cysharp.Threading.Tasks.UniTaskCompletionSourceCore`1<Cysharp.Threading.Tasks.AsyncUnit>:TrySetException (System.Exception) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTaskCompletionSource.cs:167)
Cysharp.Threading.Tasks.CompilerServices.AsyncUniTask`1<Cysharp.Threading.Tasks.UniTaskExtensions/<Timeout>d__44>:SetException (System.Exception) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/CompilerServices/StateMachineRunner.cs:210)
Cysharp.Threading.Tasks.CompilerServices.AsyncUniTaskMethodBuilder:SetException (System.Exception) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/CompilerServices/AsyncUniTaskMethodBuilder.cs:59)
Cysharp.Threading.Tasks.UniTaskExtensions/<Timeout>d__44:MoveNext () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTaskExtensions.cs:410)
Cysharp.Threading.Tasks.CompilerServices.AsyncUniTask`1<Cysharp.Threading.Tasks.UniTaskExtensions/<Timeout>d__44>:Run () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/CompilerServices/StateMachineRunner.cs:189)
Cysharp.Threading.Tasks.AwaiterActions:Continuation (object) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.cs:25)
Cysharp.Threading.Tasks.UniTaskCompletionSourceCore`1<System.ValueTuple`3<int, bool, bool>>:TrySetResult (System.ValueTuple`3<int, bool, bool>) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTaskCompletionSource.cs:139)
Cysharp.Threading.Tasks.UniTask/WhenAnyPromise`2<bool, bool>:TryInvokeContinuationT2 (Cysharp.Threading.Tasks.UniTask/WhenAnyPromise`2<bool, bool>,Cysharp.Threading.Tasks.UniTask`1/Awaiter<bool>&) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.WhenAny.Generated.cs:99)
Cysharp.Threading.Tasks.UniTask/WhenAnyPromise`2/<>c<bool, bool>:<.ctor>b__2_1 (object) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.WhenAny.Generated.cs:58)
Cysharp.Threading.Tasks.UniTaskCompletionSourceCore`1<object>:TrySetResult (object) (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTaskCompletionSource.cs:139)
Cysharp.Threading.Tasks.UniTask/DelayPromise:MoveNext () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/UniTask.Delay.cs:790)
Cysharp.Threading.Tasks.Internal.PlayerLoopRunner:RunCore () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/Internal/PlayerLoopRunner.cs:175)
Cysharp.Threading.Tasks.Internal.PlayerLoopRunner:Update () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/Internal/PlayerLoopRunner.cs:145)
Cysharp.Threading.Tasks.Internal.PlayerLoopRunner:Run () (at ./Library/PackageCache/com.cysharp.unitask@2.5.5/Runtime/Internal/PlayerLoopRunner.cs:104)

```

No log file called UnityWebBrowser.Engine.Cef.log present in project or at Library/UnityWebBrowser.Engine.Cef.log

**Comment by BrandonMayjeye** [comment]
More Information

If i run the command args against the program it has this issue as the output

```
./UnityWebBrowser.Engine.Cef -initial-url "https://voltstro.dev" -width 1920 -height 1080 -javascript True -background-color FFFFFFFF -log-path "/home/brandon/Documents/Mayjeye/UnderworldHustle/Library/UnityWebBrowser.Engine.Cef.log" -log-severity Info -comms-layer-name "TCP" -in-location "5555" -out-location "5556" -cache-path "/home/brandon/Documents/Mayjeye/UnderworldHustle/Library/UWBCache" -popup-action "Ignore" -local-storage True -proxy-server False
[1005/173719.694920:ERROR:icu_util.cc(223)] Invalid file descriptor to ICU data received.
Trace/breakpoint trap (core dumped)

```

**Comment by Voltstro** [comment]
Looks like the Linux build of the CEF engine is missing `icudtl.dat`.

**Comment by Voltstro** [comment]
This issue has been fixed in release 2.2.1.

**Comment by BrandonMayjeye** [comment]
You are a saint, 
Worked perfectly after updating all the packages in unity package list for this 

![image](https://github.com/user-attachments/assets/a594e848-225f-4807-ae68-a8db0719e83b)

Thank you .
